src/components/CommentForm.jsx

```diff
@@ -0,0 +1,35 @@
+import UserContext from '../context/UserContext';
+import {useState, useContext} from 'react'
+import { postComment } from '../utils/api';
+
+const CommentForm = (props) =>{
+    const {article_id, setCommentsArr, setCommentCount} = props
+    const { signedIn, loggedInUser } = useContext(UserContext)
+    const [commentBoxText, setCommentBoxText] = useState('')
+
+    const handleSubmit = (event) =>{
+        event.preventDefault();
+        setCommentBoxText('')
+        const date = new Date()
+        setCommentsArr((currComments)=>{
+            return [{body:commentBoxText, author: loggedInUser.username, votes: 0, created_at: `${date.toISOString()}`},...currComments]
+        })
+        setCommentCount((currCommentCount)=>{
+            return currCommentCount + 1
+        })
+        postComment(`articles/${article_id}/comments`, commentBoxText, loggedInUser.username)
```

Comment:
Ideally we want to have some error handling for if the Post fails - we need to notify the user that something went wrong